pibooth/plugins/picture_plugin.py

```diff
@@ -140,8 +144,9 @@ def state_print_do(self, cfg, app, events):
                     os.rename(osp.join(savedir, app.picture_filename), osp.join(forgetdir, app.picture_filename))
 
             self._reset_vars(app)
+            app.count.forgotten += 1
             app.previous_picture = self.second_previous_picture
 
             # Deactivate the print function for the backuped picture
             # as we don't known how many times it has already been printed
-            app.nbr_duplicates = cfg.getint('PRINTER', 'max_duplicates') + 1
+            app.count.duplicated = cfg.getint('PRINTER', 'max_duplicates') + 1
```

Comment:
This induces a strange behavior: each time I forgot a picture, this counter is set to 4 (3+1).

Reply:
It was done to prevent printing the previous picture (the one before the forgotten) as we can not know if the max counter was previously reached. But yes, we can find probably a better solution.